examples/src/main/java/io/github/tcdl/examples/MSBUtil.java

```diff
@@ -14,14 +14,24 @@
 import java.util.Map;
 
 /**
- * Created by rdrozdov-tc on 6/15/15.
+ * Utility to simplify using requester and responder server
  */
-public class BaseExample {
+public class MSBUtil {
```

Comment:
can we just have MsbUtil but not MSBUtil?  Also name of class is to general, why not ComponentCreatorHelper or simple RequesterCreatorHelper (as we create only Requesters here)?